src/core/reactor_backend.cc

```diff
@@ -1134,12 +1134,16 @@ try_create_uring(unsigned queue_len, bool throw_on_error) {
             IORING_FEAT_SUBMIT_STABLE
             | IORING_FEAT_NODROP;
     auto required_ops = {
-            IORING_OP_POLL_ADD,
-            IORING_OP_READ,
-            IORING_OP_WRITE,
+            IORING_OP_POLL_ADD, // linux 5.1
             IORING_OP_READV,
             IORING_OP_WRITEV,
             IORING_OP_FSYNC,
+            IORING_OP_SENDMSG,  // linux 5.3
+            IORING_OP_RECVMSG,
+            IORING_OP_READ,     // linux 5.6
+            IORING_OP_WRITE,
+            IORING_OP_SEND,
+            IORING_OP_RECV,
```

Comment:
Why are you making these required?

Reply:
because read and write are required. and both send and recv were added in the exact same linux version (5.6) when read and write were added. so we get send and recv for free..